docs/reference/mapping.asciidoc

```diff
@@ -177,19 +177,7 @@ PUT /my-index/_mapping
 [[update-mapping]]
 === Update the mapping of a field
 
-You can't change the mapping of an existing field, with the following
-exceptions:
-
-* You can add new <<properties,properties>> to an <<object,`object`>> field.
-* You can use the <<multi-fields,`field`>> mapping parameter to enable
-multi-fields.
-* You can change the value of the <<ignore-above,`ignore_above`>> mapping
-parameter.
-
-Changing the mapping of an existing field could invalidate data that's already
-indexed. If you need to change the mapping of a field, create a new index with
-the correct mappings and <<docs-reindex,reindex>> your data into that index. If
-you only want to rename a field, consider adding an <<alias, `alias`>> field.
+include::docs/reference/indices/put-mapping.asciidoc[tag=put-field-mapping-exceptions]
```

Comment:
@jtibshirani This includes the content tagged in [`put-mapping.asciidoc`](https://github.com/elastic/elasticsearch/pull/45520/commits/41bd30780d082410519dbbb25b078dafbbb827b3#diff-bf28969e6d6346d4d678c88fe990e15eR62).

Let me know what you think. If you'd prefer, I also just point users to the Put Mapping API docs. Thanks again for your input!

Reply:
Thanks for the update, this works for me. Tagged regions seem like a really nice feature, I'll keep them in mind for future situations where there's duplicated content.